Tasks/Kasinskiy/Task7/bus_log.py

```diff
@@ -0,0 +1,121 @@
+import data, copy
+from decimal import Decimal
+
+
+def get_list():
+    list_products = [i for i in data.get_products()] + ['Вернуться в меню']
+    return list_products
+
+
+def get_list_products():
+    list_products = get_list()
+    return '\n'.join([f'{i+1}. {list_products[i]}' for i in range(len(list_products))])
+
+
+def sort_tabl(tabl, key):
+    header = {0: ['наименование товара', 'код товара', 'цена', 'количество'], 1: ['модель', 'память', 'базовая частота', 'частота памяти', 'код товара', 'цена'], 2: ['модель', 'сокет', 'частота', 'количество ядер', 'код товара', 'цена'],
+    3: ['модель', 'сокет', 'чипсет', 'память', 'код товара', 'цена']}
```

Comment:
this variable should be placed on the global level instead of local since its value is some kind of reference and not supposed to be changed